include/aquamarine/backend/DRM.hpp

```diff
@@ -4,6 +4,7 @@
 #include "../allocator/Swapchain.hpp"
 #include "../output/Output.hpp"
 #include "../input/Input.hpp"
+#include <drm_mode.h>
```

Comment:
can we forward declare hdr_output_metadata? optional takes incomplete types AFAIK.